src/libm/mkrename.c

```diff
@@ -10,6 +10,13 @@
 
 #include "funcproto.h"
 
+char *escapeSpace(char *str) {
+  char *ret = malloc(strlen(str) + 10);
```

Comment:
This memory allocation is never released. you should modify the content of `str` in place, and definitely not return `ret`.

Reply:
I think this is harmless, but I added free for these variables.